src/03-control-flow.js

```diff
@@ -109,7 +138,14 @@ function aroundTheWorldGreeting(language) {}
  *  aroundTheWorldGreetingWithSwitch();
  *  //> "Hello World"
  */
-function aroundTheWorldGreetingWithSwitch(language) {}
+function aroundTheWorldGreetingWithSwitch(language) {
+  switch (language) {
+    case "german": return ("Hallo Welt");
```

Comment:
We should move these ` return `s down below their ` case ` s and indent.  Its easier to see what is returning and where.